internal/app/subpoll/poller.go

```diff
@@ -127,3 +151,20 @@ func (p *Poller) setLastPollTime() {
 	lastPoll := time.Now().UTC()
 	p.LastPoll = &lastPoll
 }
+
+func (p *Poller) postIsOldEnough(post *reddit.Post) bool {
+	var minimumPostAgeSeconds = p.Config.Reddit.MinimumPostAge
+
+	if minimumPostAgeSeconds == 0 {
+		return true
+	}
+
+	oldEnough := time.Since(post.Created.Time) > p.Config.Reddit.MinimumPostAge
```

Comment:
can't you use the `minimumPostAgeSeconds` variable here? also minimumPostAgeSeconds isn't really a valid variable name now given that it's a time.Duration?

Reply:
Resolved in 30c17e5.